### ISSUE TITLE
No Search and Replace Functionality

### ISSUE THREAD

**Issue by nvdl** [jendrikseipp/rednotebook#496]
### Versions

RedNotebook version: 21.9
OS: Ubuntu Bionic

There is no search and replace functionality as it is in general text editors (Ctrl + H in "gedit" for example).

**Comment by jendrikseipp** [comment]
I agree that this would be very nice to have. In case a Python coder reads this: pull requests are welcome :-)

**Comment by nvdl** [comment]
I will see if I can allocate some time to add that.
